client/tink_keyset_helper.go

```diff
@@ -42,3 +46,35 @@ func nameOfSupportedTinkKeyTemplates() string {
 	sort.Strings(supportedTemplates)
 	return strings.Join(supportedTemplates, "\n")
 }
+
+// checkTemplateNameAndKnoxIDForTinkKeyset checks whether knox identifier start with "tink:<tink_primitive_short_name>:".
+func checkTemplateNameAndKnoxIDForTinkKeyset(templateName string, knoxIentifier string) error {
+	templateInfo, existed := tinkKeyTemplates[templateName]
+	if !existed {
+		return errors.New("not supported keyset template. See 'knox key-templates'")
```

Comment:
recommend using fmt.Sprintf() instead of string concatenation (+) for the messages here.

Reply:
Thank you so much! I have revised my code according to your comment.